docs/content/union.md

```diff
@@ -119,7 +119,7 @@ Storage> union
 List of space separated upstreams.
 Can be 'upstreama:test/dir upstreamb:', '\"upstreama:test/space:ro dir\" upstreamb:', etc.
 Enter a string value. Press Enter for the default ("").
-upstreams>
+upstreams> C:\dir1 C:\dir2 C:\dir3
```

Comment:
I don't like this example on the local disk - can we use `mydrive:private/backup` like the example above?

Reply:
I based this on the following examples when the usage of the remote are showed.

![image](https://user-images.githubusercontent.com/973206/101665766-035abd00-3a13-11eb-87ac-0b063d996fca.png)

If you want to change this, we will need to change those examples too.

Reply:
Ah! Yes we probably should as `C:\dir1` is particularly confusing as it is a Windows path, but looks kind of like an rclone remote.

Maybe make 3 remotes for examples and stick to those?  s3:dir1 gdrive:dir2 dropbox:dir3 or something like that? (to pick 3  popular backends!)

Reply:
I've fixed this up now